src/compartment.c

```diff
@@ -179,6 +184,7 @@ comp_from_elf(char *filename, char **entry_points, size_t entry_point_count,
             new_comp->scratch_mem_size // size of scratch memory
     );
 
+    print_comp(new_comp);
```

Comment:
Debugging?

Reply:
Yup; removed in https://github.com/capablevms/CHERI-ELF-comp/pull/33/commits/1482d4175f08eec6c7ade8e61405266f79ef3bfa.